Formula/zork.rb

```diff
@@ -0,0 +1,25 @@
+class Zork < Formula
+  desc "The Dungeon modified from FORTRAN to C"
+  homepage "https://github.com/devshane/zork"
+  url "https://github.com/devshane/zork/archive/v1.0.2.tar.gz"
+  sha256 "169e1848b1e3c503591c23ad4e66ce45e1d5ae617831634e1da9c8fca659e283"
+  head "https://github.com/devshane/zork.git"
+
+  def install
+    system "make", "DATADIR=#{share}", "BINDIR=#{bin}"
+    system "make", "install", "DATADIR=#{share}", "BINDIR=#{bin}", "MANDIR=#{man}"
+  end
+
+  test do
+      test_phrase = <<-EOS.undent.chomp
```

Comment:
Why my vim config continues to subvert my efforts to use 2 spaces in `.rb` files is on my list of things to fix...